Makefile

```diff
@@ -116,6 +115,11 @@ _verify_scss_lint:
 # HACK: Use backticks rather than xargs because xargs swallow exit codes (everything becomes 1 and stoopidly scss-lint exits with 1 if warnings, 2 if errors)
 	@if [ -e .scss-lint.yml ]; then { scss-lint -c ./.scss-lint.yml `$(call GLOB,'*.scss')`; if [ $$? -ne 0 -a $$? -ne 1 ]; then exit 1; fi; $(DONE); } fi
 
+
+MSG_PUBLIC_IGNORE = "Please ignore built assets individually - globs and entire folders not allowed"
+_verify_assets:
+	@if grep -Eq '^\/public\/(.*\*|$$)' .gitignore; then (echo $(MSG_PUBLIC_IGNORE) && exit 1); fi
```

Comment:
I'm gonna abandon this PR in favour of https://github.com/Financial-Times/n-express/pull/353

As far as disruption goes, it only applies to 8 apps, which I'm happy to roll out to and will not need changing very often.